From e48c0ed80c88cfe24339cbb5be1735b2129ba6bb Mon Sep 17 00:00:00 2001
From: "Hebbar, Gururaja" <gururaja.hebbar@ti.com>
Date: Mon, 27 Feb 2012 11:56:39 +0530
Subject: [PATCH 022/175] ARM: OMAP2+: am335x: correct McASP0 pin mux detail

McASP0 AXR3 pin mux-mode was incorrect specified in mux file. This
patch corrects the same.

Signed-off-by: Hebbar, Gururaja <gururaja.hebbar@ti.com>
---
 arch/arm/mach-omap2/mux33xx.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-omap2/mux33xx.c b/arch/arm/mach-omap2/mux33xx.c
index 8df8fa9..59e51e0 100644
--- a/arch/arm/mach-omap2/mux33xx.c
+++ b/arch/arm/mach-omap2/mux33xx.c
@@ -351,7 +351,7 @@ static struct omap_mux __initdata am33xx_muxmodes[] = {
 		"mcasp0_axr1", NULL, NULL, "mcasp1_axr0",
 		NULL, NULL, NULL, "gpio3_20"),
 	_AM33XX_MUXENTRY(MCASP0_AHCLKX, 0,
-		"mcasp0_ahclkx", "mcasp0_axr3",	NULL, "mcasp1_axr1",
+		"mcasp0_ahclkx", NULL, "mcasp0_axr3", "mcasp1_axr1",
 		NULL, NULL, NULL, "gpio3_21"),
 	_AM33XX_MUXENTRY(XDMA_EVENT_INTR0, 0,
 		"xdma_event_intr0", NULL, NULL, NULL,
-- 
1.7.9.1

